File: examples/swagger-3/src/test/resources/asserts/typescript/axios/src/evt/BaseQueryEvt.ts
/* tslint:disable */
/* eslint-disable */

        import {BaseEvt} from "./BaseEvt";



export interface  BaseQueryEvt extends BaseEvt {

            /**
                *字段在java中的类型为：Integer
            **/
        querySize?: number;
            /**
                *字段在java中的类型为：Integer
            **/
        queryPage?: number;
}

File: examples/swagger-2/src/test/resources/asserts/java/spring_cloud_openfeign/src/com/wuxp/codegen/swagger2/evt/BaseQueryEvt.java
package com.wuxp.codegen.swagger2.evt;

import lombok.Data;
import lombok.experimental.Accessors;
import jakarta.validation.constraints.*;
import com.fasterxml.jackson.annotation.JsonFormat;
        import com.wuxp.codegen.swagger2.evt.BaseEvt;

    /**
        * 统一的查询对象
    **/
@Data
@Accessors(chain = true)
public class  BaseQueryEvt extends BaseEvt {

            /**
                *属性说明：查询大小，示例输入：
                *字段在java中的类型为：Integer
            **/
         Integer querySize;

            /**
                *属性说明：查询页码，示例输入：
                *字段在java中的类型为：Integer
            **/
         Integer queryPage;

}
